dbs/mongo/data/pg_to_mongo.sql

```diff
@@ -60,4 +60,59 @@ COPY (
         keypath AS "keypath.string()",
         value AS "value.string()"
     FROM _metabase_metadata
-) TO '/tmp/sample-_metabase_metadata.tsv' WITH (FORMAT CSV, HEADER TRUE, DELIMITER E'\t');
\ No newline at end of file
+) TO '/tmp/sample-_metabase_metadata.tsv' WITH (FORMAT CSV, HEADER TRUE, DELIMITER E'\t');
+
+COPY (
+    SELECT
+        id AS "id.int64()",
+        email AS "email.string()",
+        first_name AS "first_name.string()",
+        last_name AS "last_name.string()",
+        plan AS "plan.string()",
+        source AS "source.string()",
+        seats AS "seats.int32()",
+        created_at AS "created_at.date(2006-03-02 15:04:05.99999999)",
+        trial_ends_at AS "trial_ends_at.date(2006-03-02 15:04:05.99999999)",
+        canceled_at AS "canceled_at.date(2006-03-02 15:04:05.99999999)",
+        trial_converted AS "trial_converted.boolean()",
+        active_subscription AS "active_subscription.boolean()",
+        legacy_plan AS "legacy_plan.boolean()",
+        latitude AS "latitude.double()",
+        longitude AS "longitude.double()",
+        country AS "country.string()",
+    FROM accounts
+) TO '/tmp/sample-accounts.tsv' WITH (FORMAT CSV, HEADER TRUE, DELIMITER E'\t');
+
+COPY (
+    SELECT
+        id AS "id.int64()",
+        account_id AS "account_id.int64()",
+        event AS "event.string()",
+        timestamp AS "timestamp.date(2006-03-02 15:04:05.99999999)",
+        page_url AS "page_url.string()",
+        button_label AS "button_label.string()",
+        FROM analytic_events
+) TO '/tmp/sample-analytic_events.tsv' WITH (FORMAT CSV, HEADER TRUE, DELIMITER E'\t');
+
+COPY (
+    SELECT
+        id AS "id.int64()",
+        account_id AS "account_id.int64()",
+        email,
+        date_received,
+        rating,
+        rating_mapped,
+        body
```

Comment:
Are the missing types intentional?

Reply:
Nope, clearly I did a ctrl-z at some point and this changes didn't make it. I just updated the branch